config/config.go

```diff
@@ -151,7 +152,11 @@ func Init() error {
 	if viper.GetString("a-general.downloads") == "" {
 		// Find default Downloads dir
 		// This seems to work for all OSes?
-		DownloadsDir = filepath.Join(home, "Downloads")
+		downloadPath := userdirs.Download
+		if downloadPath == "" {
+			downloadPath = filepath.Join(home, "Downloads")
+		}
+		DownloadsDir = downloadPath
```

Comment:
```suggestion
		if userdirs.Download == "" {
			DownloadsDir = filepath.Join(home, "Downloads")
		} else {
			DownloadsDir = userdirs.Download
		}
```
Remove unecessary `downloadPath` variable.